--- a/wvdecrypter/wvdecrypter.cpp
+++ b/wvdecrypter/wvdecrypter.cpp
@@ -513,7 +513,7 @@
   if (!caps.hdcpLimit)
     caps.hdcpLimit = resolution_limit_;
 
-  //caps.flags |= (SSD_DECRYPTER::SSD_CAPS::SSD_SECURE_PATH | SSD_DECRYPTER::SSD_CAPS::SSD_ANNEXB_REQUIRED);
+  caps.flags |= (SSD_DECRYPTER::SSD_CAPS::SSD_SECURE_PATH | SSD_DECRYPTER::SSD_CAPS::SSD_ANNEXB_REQUIRED);
   //return;
 
   /*for (auto k : keys_)
@@ -557,6 +557,7 @@
       {
         LOG::LogF(SSDDEBUG, "Single decrypt possible");
         caps.flags |= SSD_DECRYPTER::SSD_CAPS::SSD_SINGLE_DECRYPT;
+        caps.flags |= (SSD_DECRYPTER::SSD_CAPS::SSD_SECURE_PATH | SSD_DECRYPTER::SSD_CAPS::SSD_ANNEXB_REQUIRED);
         caps.hdcpVersion = 99;
         caps.hdcpLimit = resolution_limit_;
       }
